L/LLVM/common.jl

```diff
@@ -353,6 +353,9 @@ rm -vrf ${prefix}/lib/*LLVM*.a
 rm -vrf ${prefix}/lib/libclang*.a
 rm -vrf ${prefix}/lib/clang
 rm -vrf ${prefix}/lib/mlir
+
+# Symlink `lld` from tools/ to bin/
+ln -s ${prefix}/bin/lld ${prefix}/tools/lld
```

Comment:
No we can't do that. The point that these binaries should not be in `bin`.

Reply:
Ok dropped that commit. Now I'm moving them back to `bin/`.

Reply:
Damnit, misread. Will move lld to `tools/`